New version: Radiomics v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: Radiomics
- Repository: https://github.com/pzaffino/Radiomics.jl
- Created by: @pzaffino
- Version: v1.0.0
- Commit: 6667ec0de19084856c49e7f687d545a13ae21c00
- Reviewed by: @pzaffino
- Reference: https://github.com/pzaffino/Radiomics.jl/commit/6667ec0de19084856c49e7f687d545a13ae21c00#commitcomment-172513298
- Description: Julia library for radiomics
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
## Breaking changes:
The BREAKING changes are:
### Changed
- First-order, shape (2D and 3D) and texture features available.
### Added
- Official website

## What's Changed (detailed):
* Feature add gldm, glrlm, glszm, ngtdm support by @jakubMitura14 in https://github.com/pzaffino/Radiomics.jl/pull/6
* Add glcm feature by @Aldogiulianicz in https://github.com/pzaffino/Radiomics.jl/pull/8
* corrections by @jakubMitura14 in https://github.com/pzaffino/Radiomics.jl/pull/9
* Insert the possibility to choise a n_bins and general optimization of code by @Aldogiulianicz in https://github.com/pzaffino/Radiomics.jl/pull/10
* official website implemented (www.radiomicsjl.org)

## New Contributors:
* @jakubMitura14 made their first contribution in https://github.com/pzaffino/Radiomics.jl/pull/6
* @Aldogiulianicz made their first contribution in https://github.com/pzaffino/Radiomics.jl/pull/8


[noblock]
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd65128dfa74fbf6ec5e5baf3135287cb3ad405065d9c5db65a0ac5413d330146cc80c1217503159d4c7db3401c0d37d3bdb25507bec5fb5d6bd0183b92094d25bf3bf77b3ce03c02fa2b4940211191e1aff474a7bd544a6e031ae168fdcdccbe4d30e921e9aa842468cca5614e898220cc5b050c202a0b3c04ae14b4178d3daf984bf11a1e84e488ce3592669604831dc18f1b7b845fb3c5eaf645e66ea7a0c304a6c773678edd20d8028bdd3dd7c26cef -->

pzaffino commented:
@JuliaRegistrator register

Release notes:

## Breaking changes:
The BREAKING changes are:
### Changed
- First-order, shape (2D and 3D) and texture features available.
### Added
- Official website

## What's Changed (detailed):
* Feature add gldm, glrlm, glszm, ngtdm support by @jakubMitura14 in https://github.com/pzaffino/Radiomics.jl/pull/6
* Add glcm feature by @Aldogiulianicz in https://github.com/pzaffino/Radiomics.jl/pull/8
* corrections by @jakubMitura14 in https://github.com/pzaffino/Radiomics.jl/pull/9
* Insert the possibility to choise a n_bins and general optimization of code by @Aldogiulianicz in https://github.com/pzaffino/Radiomics.jl/pull/10
* official website implemented (www.radiomicsjl.org)

## New Contributors:
* @jakubMitura14 made their first contribution in https://github.com/pzaffino/Radiomics.jl/pull/6
* @Aldogiulianicz made their first contribution in https://github.com/pzaffino/Radiomics.jl/pull/8


[noblock]

pzaffino commented:
@goerz any suggestion about this merge failure?

pzaffino commented:
Still the same problem @goerz (thanks for the support)

pzaffino commented:
@JuliaRegistrator register

JuliaRegistrator committed:
New version: Radiomics v1.0.0

UUID: a80a7044-09a6-4a0a-a486-eb806178b9ba
Repo: https://github.com/pzaffino/Radiomics.jl.git
Tree: da8d6b83e55ac9cccbfde04e05add9f1de69edb9

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
